client/jetpack-connect/jetpack-connect-notices.jsx

```diff
@@ -107,14 +105,6 @@ export class JetpackConnectNotices extends Component {
 				noticeValues.text = translate( 'Good news! WordPress.com sites already have Jetpack.' );
 				return noticeValues;
 
-			case IS_DOT_COM_GET_SEARCH:
```

Comment:
Shouldn't we be keeping this until it is available in production?

Reply:
makes sense